python/src/gen3authz/client/arborist/base.py

```diff
@@ -215,6 +215,10 @@ def patch(self, url, data=None, **kwargs):
     def delete(self, url, **kwargs):
         return self.request("delete", url, **kwargs)
 
+    @maybe_sync
+    async def get_users(self, url, params=None, **kwargs):
+        return await self.get(url=url, params=params, **kwargs)
```

Comment:
change `get_users` to call `_user_url` directly (and remove references to `arborist_client._user_url` from Fence)